platforms/jvm/testing-jvm/src/integTest/groovy/org/gradle/testing/testng/TestNGAssumptionFailureIntegrationTest.groovy

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.testing.testng
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.integtests.fixtures.DefaultTestExecutionResult
+
+class TestNGAssumptionFailureIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
❓ What does parameterized tests' output look like when an assumption is invalid for one particular value?  (In any engine, not just TestNG.). It might be worth adding a separate test for them - I imagine this is a common use case for assumptions: "Test everything in this range, but skip '3' because I know there's an issue with it".

It would be pretty sweet if parameters' values were automatically captured in the exception when there are invalid assumptions in a parameterized test.

Reply:
I'm not sure, but I think we treat each parameterized test as a separate test vs something combined. I can add something for this.